.electron/main.js

```diff
@@ -0,0 +1,57 @@
+// Modules to control application life and create native browser window
+const {app, BrowserWindow} = require('electron');
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let mainWindow;
+
+function createWindow () {
+  // Create the browser window.
+  mainWindow = new BrowserWindow({
+    width: 1280,
+    height: 720,
+    devTools: false,
+    webPreferences: {
+      zoomFactor: 0.8
```

Comment:
I think this is the DPI @benoitvidis is talking about: why not `1`?

Reply:
Because, we need to scroll down to fill form pages. But I think have found a solution. I'll put the PR on WIP.

Reply:
it looks more like a CSS issue than something we should tweak on Electron, no?

Reply:
Yes, you're right :disappointed: